azure-functions/acbs-function/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ const baseParserOptions = {
 };
 
 const baseRules = {
+  'no-param-reassign': ['error', { props: true, ignorePropertyModificationsForRegex: ['req', 'res'] }],
```

Comment:
What's the for this addition? I can't seem to find other instances of this in other eslint configs that seem to be aligned